core/storage/opportunity/gae_models.py

```diff
@@ -80,7 +89,12 @@ def get_export_policy(cls):
 
     @classmethod
     def get_all_translation_opportunities(
-            cls, page_size, urlsafe_start_cursor, language_code):
+            cls,
+            page_size, # type: int
+            urlsafe_start_cursor, # type: Optional[Text]
+            language_code # type: Text
+    ):
+        # type: (...) -> Tuple[Optional[List[ExplorationOpportunitySummaryModel]], Optional[Text], bool]
```

Comment:
Why is the return list optional? I think in other places we make it non optional.

Reply:
Yes I'm wrong here. fetch_page() returns empty list for no result. Thanks!